nixos/modules/programs/ssh.nix

```diff
@@ -190,8 +212,8 @@ in
         ForwardX11 ${if cfg.forwardX11 then "yes" else "no"}
 
         # Allow DSA keys for now. (These were deprecated in OpenSSH 7.0.)
```

Comment:
This comment needs to be moved. But we should think about dropping DSA support by default.